bootstrap/kubeadm/api/v1alpha3/conversion.go

```diff
@@ -75,3 +76,14 @@ func Convert_v1alpha3_KubeadmConfigStatus_To_v1alpha4_KubeadmConfigStatus(in *Ku
 	// KubeadmConfigStatus.BootstrapData has been removed in v1alpha4 because its content has been moved to the bootstrap data secret, value will be lost during conversion.
 	return autoConvert_v1alpha3_KubeadmConfigStatus_To_v1alpha4_KubeadmConfigStatus(in, out, s)
 }
+
+func Convert_v1alpha4_ClusterConfiguration_To_v1beta1_ClusterConfiguration(in *kubeadmbootstrapv1alpha4.ClusterConfiguration, out *kubeadmbootstrapv1beta1.ClusterConfiguration, s apiconversion.Scope) error {
+	// DNS.Type does was removed in v1alpha4 because only CoreDNS is supported; the information will be left to empty (kubeadm defaults it to CoredDNS);
```

Comment:
```suggestion
	// DNS.Type was removed in v1alpha4 because only CoreDNS is supported; the field will be left empty (kubeadm defaults it to CoreDNS);
```

Reply:
tnx :D